app/services/api/external-api/streaming/streaming.ts

```diff
@@ -21,8 +21,10 @@ enum EStreamingState {
 enum ERecordingState {
   Offline = 'offline',
   Starting = 'starting',
+  Start = 'start',
```

Comment:
Can we add a comment here, as to how `start` and `stop` are different?

Reply:
Done